python/lsst/ctrl/oods/gen3ButlerIngester.py

```diff
@@ -91,6 +91,39 @@ def createButler(self):
 
         return butler
 
+    def extract_info(self, dataId, key):
+        if key in dataId:
```

Comment:
I think this can be a one-liner:
```
str(dataId.get(key, "??"))
```
if you even need the `str()`.

Reply:
Fixed.